.github/CODEOWNERS

```diff
@@ -1,4 +1,5 @@
+# Default owners for everything in the repo
 * @sonarsource/platform-eng-xp-squad
 
-# No review needed for documentation changes
-*.md
+# Protect the CODEOWNERS file itself
+.github/CODEOWNERS @sonarsource/platform-eng-xp-squad
```

Comment:
That is included in the first line.

Reply:
Yes but it's to protect specifically this file, in case of added permissions on the .github folder. It's explained in the audit doc